mllib/src/main/scala/org/apache/spark/ml/classification/MultilayerPerceptronClassifier.scala

```diff
@@ -88,7 +88,7 @@ private[ml] trait MultilayerPerceptronParams extends PredictorParams
    * @group expertParam
    */
   final val weights: Param[Vector] = new Param[Vector](this, "weights",
-    " Sets the weights of the model ")
+    "Weights (either initial if before training or actual on model)")
```

Comment:
No I think `Model weights. Can be returned either after training or after explicit setting` leaves a lot to be understood compared to explaining that it can be the initial weights if before training or the actual weights after training on the model. I'll update the Scaladoc on the setter to match.